src/clu_log.c

```diff
@@ -23,7 +23,7 @@
 #include <wolfclu/clu_log.h>
 
 #ifndef WOLCLU_LOG_LINE_WIDTH
-#define WOLCLU_LOG_LINE_WIDTH 80
+#define WOLCLU_LOG_LINE_WIDTH 120
```

Comment:
I should have commented this, but 80 was chosen to match up with old school terminals and the default widths on many. If possible we should try to not go over 80 for readability on some terminals.

Reply:
I'm in the process of making WOLFSSL_ERROR improvements on the wolfSSL side, and I noticed that 80 isn't enough to print all error reasons. That's why I increased it. Would prefer to increase, as I'm skeptical anyone is going to be using wolfCLU on such an old terminal.